### PR TITLE
Show Java Version used on job

### PR DESCRIPTION
**Pull request by kuisathaverat**
@reviewbybees

### CODE REVIEW

**Comment by imod** [comment]
waiting for @reviewbybees ....

File: scriptler/showJavaVersionUseOnJob.groovy [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "name" : "Show Java Version of job",
+  "comment" : "Show Java Version used on job",
+  "parameters" : [],
+  "core": "1.609",
+  "authors" : [
+    { name : "kuisathaverat" }
+  ]
+} END META**/
+
+import jenkins.model.Jenkins
```
Comment (Evildethow) [inline scriptler/showJavaVersionUseOnJob.groovy]: pre-imported

File: scriptler/showJavaVersionUseOnJob.groovy [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import hudson.model.AbstractProject
```
Comment (Evildethow) [inline scriptler/showJavaVersionUseOnJob.groovy]: pre-imported

File: scriptler/showJavaVersionUseOnJob.groovy [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import com.cloudbees.hudson.plugins.folder.Folder
+import hudson.triggers.TimerTrigger
```
Comment (Evildethow) [inline scriptler/showJavaVersionUseOnJob.groovy]: Unused import

File: scriptler/showJavaVersionUseOnJob.groovy [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Jenkins jenkins = Jenkins.instance
+
+findAllItems(jenkins.items)
+
+def findAllItems(items){
+  for(item in items)
```
Comment (Evildethow) [inline scriptler/showJavaVersionUseOnJob.groovy]: I've already mentioned the functional alternatives enough :) You get the idea. Again, just a comment.

**Comment by Evildethow** [comment]
LGTM 🐝

**Comment by Dohbedoh** [comment]
🐝

**Comment by kuisathaverat** [comment]
@reviewbybees done

**Pull request merged**